assignments/array-methods.js

```diff
@@ -54,30 +56,75 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  
+// Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(element){
+    fullName.push(element["first_name"] + ' ' + element["last_name"]);
+  });
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
-// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+// The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. 
+// Convert each first name into all caps and log the result
 let allCaps = [];
+const drunkDirector = runners.map(function(currentValue) {
```

Comment:
```suggestion
const drunkDirector = runners.map(function(currentValue) {
    return currentValue.first_name.toUpperCase()
}

console.log(drunkDirector)
```
With .map it will return a new array, when you return a value in the .map method it will be put in the new array that .map returns, so when .map is finished that new array is returned and stored in drunkDirector

Reply:
This is just another way to solve it using .map to it's full potential